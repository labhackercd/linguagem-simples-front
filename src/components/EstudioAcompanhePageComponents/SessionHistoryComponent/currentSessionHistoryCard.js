import React from "react";
import Box from "@material-ui/core/Box";
import Grid  from "@material-ui/core/Grid";
import Typography  from "@material-ui/core/Typography";
import Paper from '@material-ui/core/Paper'
import { makeStyles } from '@material-ui/core/styles';
import StartedIcon from './assets/session_started.svg'
import 'date-fns';
import FormatStringData from './utils';
import FormatListBulletedIcon from '@material-ui/icons/FormatListBulleted';


const useStyles = makeStyles({
  root: {
    background:'#007E5A',
    border: 0,
    height: '100%',
    width: '100%',
    position: "fixed",
    display: "flex"
  },
  sessionList:{
    overflow: 'auto',
    maxHeight: 350,
    maxWidth: '100%'
  },
  input:{
      color: "green"
  },
  sessionHistoryCard:{
    background:'#F4F4F4',
  },
  currentSessionHistoryCard:{
    background:'#00AF82',
    color: 'white',
  },
  iconCurrentSessionHistoryCard:{
    background:'#007E5A',
    margin:5
  },
  iconSessionHistoryCard:{
    background:'#FFFFFF',
    margin:5
  },
  boxPaper:{
      padding:"5px"
  }
});



export default function CurrentSessionHistoryCard(props){
    const classes = useStyles();
    const sessionDate = FormatStringData(props.infoSession.date);

    return (
        <Paper elevation={0} className={classes.currentSessionHistoryCard}>
            <Grid container>
                <Grid item xs={2}>
                    <Paper elevation={0} className={classes.iconCurrentSessionHistoryCard}>
                        <Box display="flex" alignItems="center" justifyContent="center" className={classes.boxPaper}>
                            <FormatListBulletedIcon fontSize="large" style={{ color: "#FFFFFF" }}></FormatListBulletedIcon>
                        </Box>
                    </Paper>
                </Grid>
                <Grid item xs={10}>
                    <Box m={1}>
                        <Grid container>
                            <Grid item xs={6}>
                                <Typography variant="body1">{sessionDate}</Typography>
                            </Grid>
                            <Grid item xs={6}>
                                <Box display="flex" justifyContent="flex-end">
                                    <img src={StartedIcon} alt="Status sessão iniciada"></img>
                                </Box>
                            </Grid>
                            <Grid item xs={12}>
                                <Typography variant="h6">Sessão Deliberativa Extraordinária</Typography>
                            </Grid>
                        </Grid>
                    </Box>
                </Grid>
            </Grid>
        </Paper>
    );
}