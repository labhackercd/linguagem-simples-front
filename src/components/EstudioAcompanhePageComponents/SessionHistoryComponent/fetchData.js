import axiosInstance from '../../../auth/axiosApi.js'
import {API_SESSIONS_URL_NOT_LIVE, API_SESSIONS_CURRENT_LIVE_URL} from '../../../api_urls'

export async function fetchData() {

  const response = await axiosInstance.get(API_SESSIONS_URL_NOT_LIVE);
  
  //console.log("CODE: " +response.status)
  
  if(response.status===200){
    return response.data;
  }else{
    return null;
  }

};


export async function fetchCurrentSessionsData() {

  const response = await axiosInstance.get(API_SESSIONS_CURRENT_LIVE_URL);
  //console.log(response)
  
  if(response.data.length > 0){
    return response.data;
  }else{
    return null;
  }

};