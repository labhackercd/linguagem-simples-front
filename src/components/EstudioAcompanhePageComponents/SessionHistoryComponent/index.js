import React, { useState, useEffect } from "react";
import Box from "@material-ui/core/Box";
import Grid  from "@material-ui/core/Grid";
import Typography  from "@material-ui/core/Typography";
import Paper from '@material-ui/core/Paper'
import { makeStyles } from '@material-ui/core/styles';
import { withStyles } from '@material-ui/core/styles';
import List from '@material-ui/core/List';
import ListItem from '@material-ui/core/ListItem';
import AccessTimeIcon from '@material-ui/icons/AccessTime';
import SvgIcon from '@material-ui/core/SvgIcon';
import FormatListBulletedIcon from '@material-ui/icons/FormatListBulleted';
import { ReactComponent as DocumentIcon } from './assets/document_icon.svg';
import ScheduleIcon from './assets/schedule_session.svg'
import FinishedIcon from './assets/finished_session.svg'
import StartedIcon from './assets/session_started.svg'
import SessionHappening from './assets/session_happening_icon.svg'
import CircularProgress from '@material-ui/core/CircularProgress';

import 'date-fns';
import DateFnsUtils from '@date-io/date-fns';
import { MuiPickersUtilsProvider, KeyboardDatePicker,
} from '@material-ui/pickers';
import ptBrLocale from "date-fns/locale/pt-BR";

import axiosInstance from '../../../auth/axiosApi.js'


const useStyles = makeStyles({
  root: {
    background:'#007E5A',
    border: 0,
    height: '100%',
    width: '100%',
    position: "fixed",
    display: "flex"
  },
  sessionList:{
    overflow: 'auto',
    maxHeight: 350,
    maxWidth: '100%'
  },
  input:{
      color: "green"
  },
  sessionHistoryCard:{
    background:'#F4F4F4',
  },
  currentSessionHistoryCard:{
    background:'#00AF82',
    color: 'white',
  },
  iconCurrentSessionHistoryCard:{
    background:'#007E5A',
    margin:5
  },
  iconSessionHistoryCard:{
    background:'#FFFFFF',
    margin:5
  },
  boxPaper:{
      padding:"5px"
  }
});


function ScheduleOrFinishedSessionHistoryCard(props){
    const classes = useStyles();
    const sessionStatus = props.status;

    return (
        <Paper elevation={0} className={classes.sessionHistoryCard}>
            <Grid container>
                <Grid item xs={2}>
                    <Paper elevation={0} className={classes.iconSessionHistoryCard}>
                        <Box display="flex" alignItems="center" justifyContent="center" className={classes.boxPaper}>                           
                            {sessionStatus === "pre_session" ? 
                                <AccessTimeIcon fontSize="large" style={{ color: "FAC915" }}></AccessTimeIcon>
                                : 
                                <SvgIcon fontSize="large"><DocumentIcon></DocumentIcon></SvgIcon>
                            }
                        </Box>
                    </Paper>
                </Grid>
                <Grid item xs={10}>
                    <Box m={1}>
                        <Grid container>
                            <Grid item xs={6}>
                                <Typography variant="body1">17/07</Typography>
                            </Grid>
                            <Grid item xs={6}>
                                <Box display="flex" justifyContent="flex-end">
                                {sessionStatus === "pre_session" ? 
                                    <img src={ScheduleIcon} alt="Status sessão agendada"></img>
                                    : 
                                    <img src={FinishedIcon} alt="Status sessão encerrada"></img>
                                }
                                    
                                </Box>
                            </Grid>
                            <Grid item xs={12}>
                                <Typography variant="h6" style={{ color: "grey" }}>Sessão Deliberativa Extraordinária</Typography>
                            </Grid>
                        </Grid>
                    </Box>
                </Grid>
            </Grid>
        </Paper>
    );
}

function CurrentSessionHistoryCard(props){
    const classes = useStyles();

    return (
        <Paper elevation={0} className={classes.currentSessionHistoryCard}>
            <Grid container>
                <Grid item xs={2}>
                    <Paper elevation={0} className={classes.iconCurrentSessionHistoryCard}>
                        <Box display="flex" alignItems="center" justifyContent="center" className={classes.boxPaper}>                           
                            <FormatListBulletedIcon fontSize="large" style={{ color: "#FFFFFF" }}></FormatListBulletedIcon>
                        </Box>
                    </Paper>
                </Grid>
                <Grid item xs={10}>
                    <Box m={1}>
                        <Grid container>
                            <Grid item xs={6}>
                                <Typography variant="body1">17/07</Typography>
                            </Grid>
                            <Grid item xs={6}>
                                <Box display="flex" justifyContent="flex-end">
                                    <img src={StartedIcon} alt="Status sessão agendada"></img>
                                </Box>
                            </Grid>
                            <Grid item xs={12}>
                                <Typography variant="h6">Sessão Deliberativa Extraordinária</Typography>
                            </Grid>
                        </Grid>
                    </Box>
                </Grid>
            </Grid>
        </Paper>
    );
}


class SessionHistoryComponent extends React.Component {
    constructor(props){
        super(props);
        this.state = {
            sessionsList: [],
            dataLoaded:false,
        };

    }

    loadSessions(callback){
        axiosInstance.get('/sessions/', {
        }).then(
            result => {
                console.log(result)
                console.log(result.data)
                if(result.status===200){
                    this.setState({sessionsList:result.data})
                }else{
                    alert("Erro ao recuperar sessões");
                }
                callback()
            } 
        ).catch (error => {
            throw error;
        })
    }

    componentDidMount(){
        this._isMounted = true;

        if(this._isMounted){
            this.loadSessions(() => {
                console.log("Data:" + this.state.sessionsList)
                this.setState({dataLoaded:true});
 
            });
        }
    }


    render(){
        const { classes } = this.props;

        if(!this.state.dataLoaded){
            return (<Box display="flex" justifyContent="center" alignItems="center">
                        <CircularProgress></CircularProgress>
                    </Box>)
        }

        return(
            <Box>
                <Grid container>
                    <Grid item xs={12}>
                            <Grid container>
                                <Grid item xs={6}>
                                    <Box display="flex" justifyContent="flex-start" paddingBottom={3}>
                                        <Typography variant="h4" color="textSecondary">Histórico</Typography>
                                    </Box>
                                </Grid>
                                <Grid item xs={6}>
                                    <Box display="flex" justifyContent="flex-end" paddingBottom={3}>
                                        <MuiPickersUtilsProvider utils={DateFnsUtils} locale={ptBrLocale}>
                                        <KeyboardDatePicker
                                            disableToolbar
                                            variant="inline"
                                            value=""
                                            format="dd/MM/yyyy"
                                            id="date-picker-search"
                                            InputLabelProps={{
                                                shrink: true,
                                            }}
                
                                            />
                                        </MuiPickersUtilsProvider>
                                    </Box>
                                </Grid>
                            </Grid>
                    </Grid>
                    <Grid item xs={12}>
                        <Box paddingLeft={2}>
                        <img src={SessionHappening} alt="Uma sessão em andamento"></img>
                        </Box>

                        <List className={classes.sessionList} disablePadding dense={true}>
                        {this.state.sessionsList.map((session) =>
                            <ListItem key={session.id} value={session}> 
                                { (session.situation_session === "pre_session" || session.situation_session === "closed_session") ?
                                    <ScheduleOrFinishedSessionHistoryCard status={session.situation_session}></ScheduleOrFinishedSessionHistoryCard> :
                                    <CurrentSessionHistoryCard></CurrentSessionHistoryCard>
                                }
                            </ListItem>
                        )}

                        </List>
                    </Grid>
                </Grid>
                <Box pt={2}>
                    
                </Box>
            </Box>
        )
    }
}

export default withStyles(useStyles)(SessionHistoryComponent);