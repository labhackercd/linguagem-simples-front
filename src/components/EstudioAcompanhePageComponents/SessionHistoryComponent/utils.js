
export default function FormatStringData(date) {

    if(date === null || date === ""){
        return "";
    }
    var day  = date.split("-")[2];
    var month  = date.split("-")[1];
    //var year  = data.split("-")[0];
  
    return ("0"+day).slice(-2) +  '/' + ("0"+month).slice(-2) ;
    // Utilizo o .slice(-2) para garantir o formato com 2 digitos.
}