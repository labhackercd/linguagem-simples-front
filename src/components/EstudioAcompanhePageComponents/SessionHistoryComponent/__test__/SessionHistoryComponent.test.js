import React from 'react'
import {render} from '@testing-library/react';
import ReactDOM from 'react-dom'
import SessionHistoryComponent from './../index'
import renderer from 'react-test-renderer';



test('Test if SessionHistoryComponent renders without crash', () => {
    const div = document.createElement("div")
    ReactDOM.render(<SessionHistoryComponent></SessionHistoryComponent>, div)
    ReactDOM.unmountComponentAtNode(div)
});
/*
test('Test if SessionHistoryComponent snapshot matches', () => {
    const component = renderer.create(
        <SessionHistoryComponent></SessionHistoryComponent>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});



it('Test if components matches snapshot', () => {  
    const TextInputComponent = renderercreate(<NewSessionFormComponent />).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
});
*/
