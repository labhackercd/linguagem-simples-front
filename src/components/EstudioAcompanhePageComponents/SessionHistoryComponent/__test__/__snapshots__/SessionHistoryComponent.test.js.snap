// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if SessionHistoryComponent snapshot matches 1`] = `
<div
  className="MuiBox-root MuiBox-root-77"
>
  <div
    className="MuiGrid-root MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiGrid-root MuiGrid-container"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
        >
          <div
            className="MuiBox-root MuiBox-root-78"
          >
            <h4
              className="MuiTypography-root MuiTypography-h4 MuiTypography-colorTextSecondary"
            >
              Histórico
            </h4>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
        >
          <div
            className="MuiBox-root MuiBox-root-79"
          >
            <div
              className="MuiFormControl-root MuiTextField-root"
            >
              <div
                className="MuiInputBase-root MuiInput-root MuiInput-underline Mui-error Mui-error MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
                onClick={[Function]}
              >
                <input
                  aria-describedby="date-picker-search-helper-text"
                  aria-invalid={true}
                  autoFocus={false}
                  className="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
                  disabled={false}
                  id="date-picker-search"
                  onAnimationStart={[Function]}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  required={false}
                  type="text"
                />
                <div
                  className="MuiInputAdornment-root MuiInputAdornment-positionEnd"
                >
                  <button
                    className="MuiButtonBase-root MuiIconButton-root"
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    type="button"
                  >
                    <span
                      className="MuiIconButton-label"
                    >
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                        />
                        <path
                          d="M0 0h24v24H0z"
                          fill="none"
                        />
                      </svg>
                    </span>
                  </button>
                </div>
              </div>
              <p
                className="MuiFormHelperText-root Mui-error"
                id="date-picker-search-helper-text"
              >
                Invalid Date Format
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiBox-root MuiBox-root-80"
      >
        <img
          alt="Uma sessão em andamento"
          src="session_happening_icon.svg"
        />
      </div>
      <ul
        className="MuiList-root makeStyles-sessionList-70 MuiList-dense"
      >
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-currentSessionHistoryCard-73 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconCurrentSessionHistoryCard-74 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-81 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      style={
                        Object {
                          "color": "#FFFFFF",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M4 10.5c-.83 0-1.5.67-1.5 1.5s.67 1.5 1.5 1.5 1.5-.67 1.5-1.5-.67-1.5-1.5-1.5zm0-6c-.83 0-1.5.67-1.5 1.5S3.17 7.5 4 7.5 5.5 6.83 5.5 6 4.83 4.5 4 4.5zm0 12c-.83 0-1.5.68-1.5 1.5s.68 1.5 1.5 1.5 1.5-.68 1.5-1.5-.67-1.5-1.5-1.5zM7 19h14v-2H7v2zm0-6h14v-2H7v2zm0-8v2h14V5H7z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-82"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-83"
                      >
                        <img
                          alt="Status sessão agendada"
                          src="session_started.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-84 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      style={
                        Object {
                          "color": "FAC915",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                      <path
                        d="M12.5 7H11v6l5.25 3.15.75-1.23-4.5-2.67z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-85"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-86"
                      >
                        <img
                          alt="Status sessão agendada"
                          src="schedule_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-87 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      style={
                        Object {
                          "color": "FAC915",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                      <path
                        d="M12.5 7H11v6l5.25 3.15.75-1.23-4.5-2.67z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-88"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-89"
                      >
                        <img
                          alt="Status sessão agendada"
                          src="schedule_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-90 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      style={
                        Object {
                          "color": "FAC915",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                      />
                      <path
                        d="M12.5 7H11v6l5.25 3.15.75-1.23-4.5-2.67z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-91"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-92"
                      >
                        <img
                          alt="Status sessão agendada"
                          src="schedule_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-93 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <svg>
                        document_icon.svg
                      </svg>
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-94"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-95"
                      >
                        <img
                          alt="Status sessão encerrada"
                          src="finished_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-96 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <svg>
                        document_icon.svg
                      </svg>
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-97"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-98"
                      >
                        <img
                          alt="Status sessão encerrada"
                          src="finished_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-dense MuiListItem-gutters"
          disabled={false}
        >
          <div
            className="MuiPaper-root makeStyles-sessionHistoryCard-72 MuiPaper-elevation0 MuiPaper-rounded"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
              >
                <div
                  className="MuiPaper-root makeStyles-iconSessionHistoryCard-75 MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <div
                    className="MuiBox-root MuiBox-root-99 makeStyles-boxPaper-76"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <svg>
                        document_icon.svg
                      </svg>
                    </svg>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
              >
                <div
                  className="MuiBox-root MuiBox-root-100"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <p
                        className="MuiTypography-root MuiTypography-body1"
                      >
                        17/07
                      </p>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
                    >
                      <div
                        className="MuiBox-root MuiBox-root-101"
                      >
                        <img
                          alt="Status sessão encerrada"
                          src="finished_session.svg"
                        />
                      </div>
                    </div>
                    <div
                      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <h6
                        className="MuiTypography-root MuiTypography-h6"
                        style={
                          Object {
                            "color": "grey",
                          }
                        }
                      >
                        Sessão Deliberativa Extraordinária
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-102"
  />
</div>
`;
